server/channels/app/channel.go

```diff
@@ -2933,7 +2933,6 @@ func (a *App) SearchAllChannels(c request.CTX, term string, opts model.ChannelSe
 		PolicyID:                 opts.PolicyID,
 		IncludePolicyID:          opts.IncludePolicyID,
 		IncludeSearchById:        opts.IncludeSearchById,
-		ExcludeRemote:            opts.ExcludeRemote,
```

Comment:
Is this related?

Reply:
Great catch! No, not related at all. May have been an error while merging.